index.js

```diff
@@ -15,6 +18,11 @@ const githubAuthToken = {
 
 accuser.authenticate(githubAuthToken);
 
+// Greylisted
+for (let level = 1; level <= maxLevel; level += 1) {
```

Comment:
Should probably document this somewhere as the non-standard use of "WhiteList", "BlackList", "GreyList" may come off as confusing

Reply:
https://github.com/nus-cs2103/cs2103-pr-bot/blob/0695ce25716fef7f210c32712854113757c7c237/src/Greylist.js#L5

Reply:
it's there!

Reply:
I mean in the README